parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -1,28 +1,5 @@
 #!/bin/bash
 
-###########################################################
```

Comment:
This mast header text no longer relevant

Reply:
💥